src/reducers/player/index.js

```diff
@@ -56,3 +59,11 @@ function removeTrackFromPlaylist(state, trackId) {
 function emptyPlaylist(state) {
   return { ...state, playlist: [] };
 }
+
+function setShuffleMode(state) {
+  if (state.shuffleMode === false) {
+    return { ...state, shuffleMode: true };
+  } else {
+    return { ...state, shuffleMode: false };
+  }
```

Comment:
A shortcut for the whole block could be: `return { ...state, shuffleMode: !state.shuffleMode  }` You simply invert the shuffleMode.